src/kin/2022-03/01/01.md

```diff
@@ -0,0 +1,11 @@
+ITANURA RY’UMWUNGERI*
+*KU ISABATO NIMUGOROBA, 25 KAMENA 2022*
+### AHO ICYIGISHO CY’IKI CYUMWERU GISHINGIYE:* Zaburi 23, Abaroma 12:18-21
+><p>ICYO KWIBUKWA:</p>
+>“Asubiza intege mu bugingo bwanjye, anyobora inzira yo gukiranuka ku bw’izina rye”. (Zaburi 23:3).
```

Comment:
@Burkifa23 Please see en/2022-04/01/01.md to see how to properly format Sabbath day